docs/hugo/content/guide/metrics.md

```diff
@@ -29,8 +30,71 @@ By default, metrics for ASOv2 are turned on and can be toggled by the following
       containers:
        - args:
          - --metrics-addr=0.0.0.0:8080 (default)    
+         - --secure-metrics=true/false (default: false)
     ```
-   
+
+## Scraping Metrics Securely via HTTPs
+
+A ServiceAccount token is required to scrape metrics securely. The corresponding ServiceAccount needs permissions on the "/metrics" and "debug/pprof" paths. 
+This can be achieved e.g. by following the [Kubernetes documentation](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/).
+
+- Use the settings below in your deployment: 
+
+- #### ASOv2 Helm Chart
```

Comment:
I don't think this bullet point needs to be a heading - and if it were a heading, it would be 3rd level, not 4th.

Suggest changing to plain text.